### PR TITLE
Rename `histology` to `anat` and move to session level.

### PR DESCRIPTION
**Pull request by JoeZiminski**
Further to Neuroblueprint change [#299](https://github.com/neuroinformatics-unit/NeuroBlueprint/pull/46) this PR renames `histology` to `anat` and moves it to the session level.

### CODE REVIEW

File: docs/source/pages/documentation.md [hunk 1]
```diff
@@ -90,10 +90,9 @@ ssh \
 --central_host_id ssh.swc.ucl.ac.uk \
 --central_host_username username \
 --transfer_verbosity v \
---use-ephys --use-behav --use-histology --overwrite_old_files
+--use-ephys --use-behav --use-anat --overwrite_old_files
```
Comment (niksirbi) [inline docs/source/pages/documentation.md:93]: These flags need to be removed, right? My understanding was that they are no longer configured, and have to be supplied each time.

File: docs/source/pages/documentation.md [hunk 2]
```diff
@@ -142,25 +141,22 @@ When the `all` argument is used for `--datatype` (`-dt`), the folders created de
 │   ├── ses-010_time-160248/
 │   │   ├── behav
 │   │   └── funcimg
-│   └── histology
+│   └── anat
```
Comment (niksirbi) [inline docs/source/pages/documentation.md:144]: In all the example folder trees in this document, the `anat` is still shown on the subject level. Move to the session level, either together with the other datatypes, or in its own session.

File: README.md [hunk 3]
```diff
@@ -38,14 +38,14 @@ See the SWC-BIDS [specification](https://swc-bids.neuroinformatics.dev/) for mor
         │   └── ses-001/
         │   │   ├── ephys/
         │   │   └── behav/
-        │   └── histology/
```
Comment (niksirbi) [inline README.md:41]: Also here, `anat` is shown on the subject level in example folder trees.

File: tests/test_utils.py [hunk 4]
```diff
@@ -324,7 +324,7 @@ def check_datatype_sub_ses_uploaded_correctly(
         sub_names = glob_basenames(join(base_path_to_check, "*"))
         assert sub_names == sorted(subs_to_upload)
 
-        # Check ses are all uploaded + histology if transferred
+        # Check ses are all uploaded + anat if transferred
```
Comment (niksirbi) [inline tests/test_utils.py:327]: This comment is wrong now, anat is part of ses

**Review by niksirbi — CHANGES_REQUESTED** [review]
The example project trees in documentaiton and in README have to be changed so that they show `anat` at the session level. 

There is also one lingering mention of "histological" in documentation.md line 79 (should be "anatomical")

**Comment by JoeZiminski** [comment]
Thanks @niksirbi! the comments are addressed, I just realised there are some minor formatting issues with the tree connectors, but these are minor and are fixed in #207.

**Review by niksirbi — APPROVED** [review]

**Pull request merged**